test/models/transfer_test.rb

```diff
@@ -15,12 +15,12 @@
 class TransferTest < ActiveSupport::TestCase
   setup do
     @transfer = transfers(:valid)
-    file = Rails.root.join('test', 'fixtures', 'files', 'a_pdf.pdf')
-    @transfer.files.attach(io: File.open(file), filename: 'a_pdf.pdf')
+    # file = Rails.root.join('test', 'fixtures', 'files', 'a_pdf.pdf')
+    # @transfer.files.attach(io: File.open(file), filename: 'a_pdf.pdf')
   end
 
   teardown do
```

Comment:
I think we can remove this teardown method now that we're not attaching anything in the setup.